features/session_stopping.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Stopping and resuming sessions
+
+Scenario Outline: When a session is stopped the error has no session information
+    When I launch an <platform> app with "StoppedSessionScenario"
+    Then I should receive 2 requests
+    And the request 0 is valid for the session tracking API
+    And the request 1 is valid for the error reporting API
+    And the payload field "events.0.session" is null for request 1
+
+    Examples:
+    | platform |
+    | Android  |
+    | iOS      |
+
+Scenario Outline: When a session is resumed the error uses the previous session
```

Comment:
It doesn't seem to me that the assertions in this test are doing what the outline says. I would expect to see an assertions that given the following sequence of events:

```
- startSession()
- send report
- stopSession()
- send report
- resumeSession()
- send report
```

The following conditions are met:

```
- report 1 and report 3 have the same session id
- report 2 has no session
- there was one session received
- report 1 has one report in its session
- report 3 has two events in its session
```